server/models/postgis/layers.py

```diff
@@ -17,11 +18,22 @@ class Layer(db.Model):
     map_category = db.Column(db.Integer, default=0, nullable=False)
     layer_source = db.Column(db.String)
 
-    def create(self):
-        """ Creates and saves the current model to the DB """
-        db.session.add(self)
+    @classmethod
+    def create_from_dto(cls, layer_dto: LayerDetailsDTO):
+        """ Creates new layer from DTO """
+        new_layer = cls()
+        new_layer.layer_name = layer_dto.layer_name
+        new_layer.layer_title = layer_dto.layer_title
+        new_layer.layer_description = layer_dto.layer_description
+        new_layer.layer_source = layer_dto.layer_source
+        new_layer.map_category = MapCategory[layer_dto.map_category].value
```

Comment:
It isn't critical but there will be a duplicate of map_category in the response because we're sending back arrays of Preparedness, Incidents and Awareness layers, each containing the category.